run_acceptance_tests.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+cd /app
+
+pipenv install --dev
+
+OUTPUT=$(pipenv run python generate_qid_batch.py test_batch.csv)
+BATCH_ID=${OUTPUT: -36}
+
+sleep 15
```

Comment:
We could consume uac created events until we reach the expected number here instead of hardcoding a wait.